lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu
+  def initialize(items = [])
+    @items = items
+  end
+
+  def display_all
+    puts @items
+  end
+
+  def display_food
+    @items.each { |item| puts item if item.class == Food }
```

Comment:
Imagine we had a menu for a bar with 300 drinks and 1 food offering, for this code, we'd have to go through all 301 items just to print the one food item. I wonder if having separate food and drink arrays would help here?